packages/tools/addon/templates/components/cs-field-overlays.hbs

```diff
@@ -6,20 +6,25 @@
       mark.model.content
       mark.model.caption
       (eq mark.id tools.openedFieldId)
+      (eq mark.id tools.detailedViewFieldId)
       (cs-field-editor-for mark.model.content mark.model.name variant="inline")
-    as |fieldName content caption isOpenedField editorName|}}
+    as |fieldName content caption isOpenedField isDetailedViewField editorName|}}
       {{#let (is-component editorName) as |hasInlineEditor|}}
+        <div hidden=true onclick={{action "openField" mark}} data-test-overlay-button={{dasherize fieldName}}>
```

Comment:
What's a better way to do this? Trying to click on the field overlay below doesn't work during testing so I had to add a hidden `div` with the same click action on it.

Reply:
Do you have any ideas why the click doesn't work in testing?

I would add an Esc key action, for both keyboard nav and testing. In testing you'd have to still set the focus on the overlay though. See "closing the dialog" on https://www.nomensa.com/blog/2014/how-improve-accessibility-overlay-windows-part-1